packages/cli-node/api-report.md

```diff
@@ -154,6 +154,7 @@ export type PackageRole =
   | 'node-library'
   | 'common-library'
   | 'frontend-plugin'
+  | 'frontend-extension-bundle'
```

Comment:
Hmm, I was thinking `frontend-extensions`, how do you feel about that option? Don't change anything yet because I'm not too sure about it either 😅

Reply:
I'm open to that! I'm not merging this before next week either way since I'm working on adding an example package that's consumed by app-next. Let's talk then.

Reply:
The `frontend-plugin-module` role seems to actually be in use though, even in our own repo. May need to deal with that separately.